src/core/server/http_resources/http_resources_service.test.ts

```diff
@@ -85,7 +85,6 @@ describe('HttpResources service', () => {
             register(routeConfig, async (ctx, req, res) => {
               return res.renderCoreApp({
                 headers: {
-                  'content-security-policy': "script-src 'unsafe-eval'",
                   'x-kibana': '42',
```

Comment:
This header is supposed to be included so the test case can assert that it has no effect (the Core rendering service overrides it with its own header). So you should add this back in:

```suggestion
                  'content-security-policy': "script-src 'unsafe-eval'",
                  'x-kibana': '42',
```